### ISSUE TITLE
Share invitation link for location admins (+ plug-ins)

### ISSUE THREAD

**Issue by charlesrc** [cknaut/cinemaple#84]

**Comment by charlesrc** [comment]
For the plug-ins, here is a python library that can do it:
https://github.com/fcurella/django-social-share

Let's talk about adding it to the project? It provides some very simple/elegant way to add social media sharing links to any page. 

That seems like a low cost/high pay-off add-on.

**Comment by cknaut** [comment]
Yeah, let's try it out!Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: charlesrc <notifications@github.com> Date: 11/24/19  11:37  (GMT-05:00) To: cknaut/cinemaple <cinemaple@noreply.github.com> Cc: cknaut <can.knaut@gmail.com>, Assign <assign@noreply.github.com> Subject: Re: [cknaut/cinemaple] Share invitation link for location admins (+ plug-ins) (#84) For the plug-ins, here is a python library that can do it:
https://github.com/fcurella/django-social-share
Let's talk about adding it to the project? It provides some very simple/elegant way to add social media sharing links to any page.
That seems like a low cost/high pay-off add-on.

—You are receiving this because you were assigned.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/cknaut/cinemaple/issues/84?email_source=notifications\u0026email_token=ABF2QERNJDLPU63XOMNCQNTQVKUVXA5CNFSM4JHU5VX2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEFAPMKI#issuecomment-557905449",
"url": "https://github.com/cknaut/cinemaple/issues/84?email_source=notifications\u0026email_token=ABF2QERNJDLPU63XOMNCQNTQVKUVXA5CNFSM4JHU5VX2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEFAPMKI#issuecomment-557905449",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by charlesrc** [comment]
Some information :
-- You will have to merge the branch that was automatically created as a bump from django 2.2 to 2.2.4 that was created when I updated the requirements.txt
-- I created a dummy example in change_profile.html
This creates a large modal with a bunch of social media sharing.
For now Facebook does not really work, because it only works with URLs. But in the end it should be no problem since we will update to a URL sharing. 
Twitter works, email works, LinkedIn seems to be buggy, haven't tested reddit/Telegram but that doesn't sound like the most useful anyways... We can keep it to FB/email in the future.
